interceptor/tracing_interceptor_internal_test.go

```diff
@@ -0,0 +1,26 @@
+package interceptor
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"go.temporal.io/sdk/workflow"
+)
+
+func TestIdempotencyKeyGeneration(t *testing.T) {
```

Comment:
Since this key is not used currently, I support no integration test with it. May want one to confirm counter is workflow-specific and replay safe though in the impl that does use it.